Assets/MixedRealityToolkit/Common/Shaders/Obsolete/HoloToolkitCommon.cginc

```diff
@@ -1,35 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-#include "UnityCG.cginc"
```

Comment:
Ahh, looks like there were two HoloToolkitCommon.cginc files, one in the Obsolete folder and one in the FastConfigurable folder, and this one has that `#include`.

Reply:
That explains why it wasn't picked up before #Phew